.github/workflows/ci-cargo-audit.yml

```diff
@@ -32,4 +32,4 @@ jobs:
       # Run cargo audit using args from .cargo/audit.toml (ignores, etc.)
       - name: Run Cargo Audit
         # Ignoring warnings from solana_program_test dependencies which are only in dev dependencies
-        run: cargo audit -c always --ignore RUSTSEC-2021-0139 --ignore RUSTSEC-2023-0001 --ignore RUSTSEC-2020-0071
+        run: cargo audit -c always --ignore RUSTSEC-2021-0139 --ignore RUSTSEC-2023-0001 --ignore RUSTSEC-2020-0071 --ignore RUSTSEC-2023-0034
```

Comment:
Suppressing this issue that is only in solana-program-test (not for onchain code)
```
Crate:     h2
Version:   0.3.16
Title:     Resource exhaustion vulnerability in h2 may lead to Denial of Service (DoS)
Date:      2023-04-14
ID:        RUSTSEC-2023-0034
URL:       https://rustsec.org/advisories/RUSTSEC-2023-0034
Solution:  Upgrade to >=0.3.17
error: 1 vulnerability found!
Dependency tree:
h2 0.3.16
├── reqwest 0.11.14
│   ├── solana-rpc-client-api 1.15.2
│   │   ├── solana-tpu-client 1.15.2
│   │   │   ├── solana-udp-client 1.15.2
│   │   │   │   └── solana-client 1.15.2
│   │   │   │       ├── solana-send-transaction-service 1.15.2
│   │   │   │       │   └── solana-banks-server 1.15.2
│   │   │   │       │       └── solana-program-test 1.15.2
│   │   │   │       │           └── uxd 7.0.5
│   │   │   │       └── solana-banks-server 1.15.2
│   │   │   ├── solana-thin-client 1.15.2
│   │   │   │   └── solana-client 1.15.2
│   │   │   ├── solana-send-transaction-service 1.15.2
│   │   │   ├── solana-quic-client 1.15.2
│   │   │   │   └── solana-client 1.15.2
│   │   │   └── solana-client 1.15.2
│   │   ├── solana-thin-client 1.15.2
│   │   ├── solana-rpc-client 1.15.2
│   │   │   ├── solana-tpu-client 1.15.2
│   │   │   ├── solana-thin-client 1.15.2
│   │   │   ├── solana-rpc-client-nonce-utils 1.15.2
│   │   │   │   └── solana-client 1.15.2
│   │   │   └── solana-client 1.15.2
│   │   ├── solana-quic-client 1.15.2
│   │   ├── solana-pubsub-client 1.15.2
│   │   │   ├── solana-tpu-client 1.15.2
│   │   │   └── solana-client 1.15.2
│   │   └── solana-client 1.15.2
│   ├── solana-rpc-client 1.15.2
│   ├── solana-pubsub-client 1.15.2
│   └── solana-metrics 1.15.2
│       ├── solana-vote-program 1.15.2
│       │   ├── solana-stake-program 1.15.2
│       │   │   └── solana-runtime 1.15.2
│       │   │       ├── solana-send-transaction-service 1.15.2
│       │   │       ├── solana-program-test 1.15.2
│       │   │       └── solana-banks-server 1.15.2
│       │   ├── solana-runtime 1.15.2
│       │   ├── solana-rpc-client 1.15.2
│       │   ├── solana-program-test 1.15.2
│       │   └── solana-perf 1.15.2
│       │       ├── solana-streamer 1.15.2
│       │       │   ├── solana-udp-client 1.15.2
│       │       │   ├── solana-quic-client 1.15.2
│       │       │   └── solana-client 1.15.2
│       │       ├── solana-runtime 1.15.2
│       │       └── solana-clap-utils 1.15.2
│       │           └── solana-rpc-client-nonce-utils 1.15.2
│       ├── solana-tpu-client 1.15.2
│       ├── solana-streamer 1.15.2
│       ├── solana-send-transaction-service 1.15.2
│       ├── solana-runtime 1.15.2
│       ├── solana-quic-client 1.15.2
│       ├── solana-program-runtime 1.15.2
│       │   ├── solana-zk-token-proof-program 1.15.2
│       │   │   └── solana-runtime 1.15.2
│       │   ├── solana-vote-program 1.15.2
│       │   ├── solana-stake-program 1.15.2
│       │   ├── solana-runtime 1.15.2
│       │   ├── solana-program-test 1.15.2
│       │   ├── solana-config-program 1.15.2
│       │   │   ├── solana-stake-program 1.15.2
│       │   │   ├── solana-runtime 1.15.2
│       │   │   └── solana-account-decoder 1.15.2
│       │   │       ├── solana-transaction-status 1.15.2
│       │   │       │   ├── solana-rpc-client-api 1.15.2
│       │   │       │   └── solana-rpc-client 1.15.2
│       │   │       ├── solana-rpc-client-api 1.15.2
│       │   │       ├── solana-rpc-client 1.15.2
│       │   │       └── solana-pubsub-client 1.15.2
│       │   ├── solana-compute-budget-program 1.15.2
│       │   │   └── solana-runtime 1.15.2
│       │   ├── solana-bpf-loader-program 1.15.2
│       │   │   ├── solana-runtime 1.15.2
│       │   │   └── solana-program-test 1.15.2
│       │   └── solana-address-lookup-table-program 1.15.2
│       │       ├── solana-transaction-status 1.15.2
│       │       ├── solana-runtime 1.15.2
│       │       └── solana-account-decoder 1.15.2
│       ├── solana-perf 1.15.2
│       ├── solana-connection-cache 1.15.2
│       │   ├── solana-udp-client 1.15.2
│       │   ├── solana-tpu-client 1.15.2
│       │   ├── solana-thin-client 1.15.2
│       │   ├── solana-quic-client 1.15.2
│       │   └── solana-client 1.15.2
│       └── solana-client 1.15.2
└── hyper 0.14.24
    ├── reqwest 0.11.14
    └── hyper-rustls 0.23.2
        └── reqwest 0.11.14
```

Reply:
Can add as comment in the code maybe for posterity

Reply:
Yeah, I think same as the other exceptions in there, did you mean we should add something else than the comment right above on line 34? or another file we should add this to?